### ISSUE TITLE
Option summarizeFailureOutput: true does not work with emitError: false

### ISSUE THREAD

**Issue by tkrotoff** [panuhorsmalahti/gulp-tslint#41]
They should be independent.

Example:
```TypeScript
gulp.task('tslint', () =>
    ...
    .pipe(tslint.report('prose', {emitError: false, summarizeFailureOutput: true}))
);
```

```Shell
$ gulp tslint
...
[22:29:42] [gulp-tslint] error app.ts[24, 71]: missing whitespace
[22:29:43] Finished 'tslint' after 865 ms
```

vs

```TypeScript
gulp.task('tslint', () =>
    ...
    .pipe(tslint.report('prose', {emitError: true, summarizeFailureOutput: true}))
);
```

```Shell
$ gulp tslint
...
[22:29:01] [gulp-tslint] error app.ts[24, 71]: missing whitespace
[22:29:01] 'tslint' errored after 851 ms
[22:29:01] Error in plugin 'gulp-tslint'
Message:
    Failed to lint: 1 errors.
```

Would be nice to have the message `Failed to lint: x errors` even if `emitError` is false.
In my case it helps me to see if the number of errors increases (knowing that I have already n errors that are false positive).

**Comment by panuhorsmalahti** [comment]
Fixed in 4.1.0.

**Comment by tkrotoff** [comment]
Cool! Thx
